lib/routing.ts

```diff
@@ -2,11 +2,26 @@ import { PerCollection } from "./types/perCollection";
 
 export type PerCollectionCodenames = PerCollection<string | null>;
 
-export const pageCodenames = {
+// Those are used for static site generation in pages/[slug].tsx
```

Comment:
I don't really like having the definition (of what the particular page `pages/[slug].tsx` renders) defined here. What about having the set of keys (in this case just `["about-us"]` defined in the `pages/[slug].tsx` so it can pick only the relevant keys from the `pageCodenames` and then we can inline the `pageCodenamesForGenericPrerender` into `pageCodenames`? I am afraid that this file would grow with weird names for all the different pages that would want some "special" codenames.